content/pt-br/docs/concepts/policy/resource-quotas.md

```diff
@@ -0,0 +1,649 @@
+---
+title: Resource Quotas
```

Comment:
It would be better to translate "Resource Quotas" in here too!

Reply:
Done!